DirectionsPlayground/DirectionsPlayground/Sources/InfoButton.swift

```diff
@@ -0,0 +1,38 @@
+import SwiftUI
+
+struct InfoButton: View {
+    enum DocUrl {
+        case waypointAllowsSnappingToClosedRoad
+        case waypointAllowsArrivingOnOppositeSide
+        case separatesLegs
+    }
+
+    let docUrl: DocUrl
+    @Environment(\.openURL) var openURL
+
+    var body: some View {
+        Button {
+            openURL(docUrl.url)
+        } label: {
+            Image(systemName: "info.circle")
+        }
+        .buttonStyle(.bordered)
+    }
+}
+
+extension InfoButton.DocUrl {
+    var urlString: String {
+        switch self {
+        case .waypointAllowsArrivingOnOppositeSide:
+            return "https://docs.mapbox.com/ios/directions/api/2.3.0/Classes/Waypoint.html#/s:16MapboxDirections8WaypointC28allowsArrivingOnOppositeSideSbvp"
```

Comment:
It looks like we would have to update the version here with each Directions SDK release.

Should we use 2.8.0 here for now?